src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -382,6 +386,16 @@ export default class CoExpressionTab extends React.Component<
                                         entrezGeneIdToReferenceGene[
                                             datum.geneticEntityId
                                         ].cytoband;
+                                } else if (
+                                    entrezGeneIdToGene[
+                                        datum.geneticEntityId
+                                    ] !== undefined
+                                ) {
+                                    geneticEntityName =
+                                        entrezGeneIdToGene[
+                                            datum.geneticEntityId
+                                        ].hugoGeneSymbol;
+                                    cytoband = '';
```

Comment:
Is it ok to assign empty string to `cytoband`?
```
cytoband = '';
```

Reply:
Good question. It looks empty in the tutorial video https://www.youtube.com/watch?v=YUxVv6pkxD4&t=2240s, but that may not always be the case.